commit da55fae9e277a0c138d4395fee505e5d2f8b2b84
Author: Carlos O'Donell <carlos@redhat.com>
Date:   Tue May 23 08:34:56 2023 -0400

    support: Reformat Makefile.
    
    Add list end markers.
    Sort text using scripts/sort-makefile-lines.py.
    
    No code generation changes observed in non-test binary artifacts.
    No regressions on x86_64 and i686.

Conflicts:
	support/Makefile
	  (different backport order)


diff --git a/support/Makefile b/support/Makefile
index 0273c0f6306720c9..17a4157563f9ecd6 100644
--- a/support/Makefile
+++ b/support/Makefile
@@ -41,7 +41,7 @@ libsupport-routines = \
   resolv_response_context_free \
   resolv_test \
   set_fortify_handler \
-  support_stack_alloc \
+  support-open-dev-null-range \
   support_become_root \
   support_can_chroot \
   support_capture_subprocess \
@@ -65,12 +65,11 @@ libsupport-routines = \
   support_fuse \
   support_isolate_in_subprocess \
   support_need_proc \
+  support_openpty \
   support_path_support_time64 \
+  support_paths \
   support_process_state \
   support_ptrace \
-  support-open-dev-null-range \
-  support_openpty \
-  support_paths \
   support_quote_blob \
   support_quote_blob_wide \
   support_quote_string \
@@ -84,6 +83,8 @@ libsupport-routines = \
   support_shared_allocate \
   support_small_stack_thread_attribute \
   support_socket_so_timestamp_time64 \
+  support_stack_alloc \
+  support_stack_alloc \
   support_stat_nanoseconds \
   support_subprocess \
   support_test_compare_blob \
@@ -96,11 +97,11 @@ libsupport-routines = \
   support_write_file_string \
   temp_file \
   timespec \
-  timespec-time64 \
   timespec-add \
   timespec-add-time64 \
   timespec-sub \
   timespec-sub-time64 \
+  timespec-time64 \
   write_message \
   xaccept \
   xaccept4 \
@@ -108,6 +109,8 @@ libsupport-routines = \
   xbind \
   xcalloc \
   xchdir \
+  xchmod \
+  xchmod \
   xchroot \
   xclock_gettime \
   xclock_gettime_time64 \
@@ -116,7 +119,6 @@ libsupport-routines = \
   xclone \
   xclose \
   xclosedir \
-  xchmod \
   xconnect \
   xcopy_file_range \
   xdlfcn \
@@ -147,11 +149,17 @@ libsupport-routines = \
   xpipe \
   xpoll \
   xposix_memalign \
+  xposix_spawn \
+  xposix_spawn \
+  xposix_spawn_file_actions_addclose \
+  xposix_spawn_file_actions_addclose \
+  xposix_spawn_file_actions_adddup2 \
+  xposix_spawn_file_actions_adddup2 \
   xpthread_attr_destroy \
   xpthread_attr_init \
+  xpthread_attr_setaffinity_np \
   xpthread_attr_setdetachstate \
   xpthread_attr_setguardsize \
-  xpthread_attr_setaffinity_np \
   xpthread_attr_setstack \
   xpthread_attr_setstacksize \
   xpthread_barrier_destroy \
@@ -162,8 +170,8 @@ libsupport-routines = \
   xpthread_barrierattr_setpshared \
   xpthread_cancel \
   xpthread_check_return \
-  xpthread_cond_wait \
   xpthread_cond_signal \
+  xpthread_cond_wait \
   xpthread_create \
   xpthread_detach \
   xpthread_join \
@@ -182,8 +190,8 @@ libsupport-routines = \
   xpthread_mutexattr_setrobust \
   xpthread_mutexattr_settype \
   xpthread_once \
-  xpthread_rwlock_init \
   xpthread_rwlock_destroy \
+  xpthread_rwlock_init \
   xpthread_rwlock_rdlock \
   xpthread_rwlock_unlock \
   xpthread_rwlock_wrlock \
@@ -204,9 +212,6 @@ libsupport-routines = \
   xsigstack \
   xsocket \
   xstatx \
-  xposix_spawn \
-  xposix_spawn_file_actions_addclose \
-  xposix_spawn_file_actions_adddup2 \
   xstrdup \
   xstrndup \
   xsymlink \
@@ -215,6 +220,7 @@ libsupport-routines = \
   xuselocale \
   xwaitpid \
   xwrite \
+  # libsupport-routines
 
 libsupport-static-only-routines := $(libsupport-routines)
 # Only build one variant of the library.
@@ -278,8 +284,16 @@ LDLIBS-test-container = $(libsupport)
 others += test-container
 others-noinstall += test-container
 
-others += shell-container echo-container true-container
-others-noinstall += shell-container echo-container true-container
+others += \
+  echo-container \
+  shell-container \
+  true-container \
+  # others
+others-noinstall += \
+  echo-container \
+  shell-container \
+  true-container \
+  # others-noinstall
 
 others += $(LINKS_DSO_PROGRAM)
 others-noinstall += $(LINKS_DSO_PROGRAM)
@@ -317,6 +331,7 @@ tests = \
   tst-xdirent \
   tst-xreadlink \
   tst-xsigstack \
+  # tests
 
 ifeq ($(run-built-tests),yes)
 tests-special = \
